tools/protoc-plugin/src/main/java/io/spine/tools/protoc/column/ColumnGenerator.java

```diff
@@ -45,7 +45,7 @@ private ColumnGenerator(boolean generate) {
      */
     public static ColumnGenerator instance(SpineProtocConfig config) {
```

Comment:
Should this class still be called `ColumnGenerator`?

Reply:
@dmdashenkov yes, we still generate the columns independently from the queries, as previously. We just don't have an option to generate _only_ the columns.